totest-manager.py

```diff
@@ -1016,6 +1016,20 @@ class ToTest151Images(ToTest150Images):
     def openqa_group(self):
         return 'openSUSE Leap 15.1 Images'
 
+class ToTest151ARMImages(ToTest150Images):
```

Comment:
Should inheriting ToTest151Images

Reply:
Could, but it's also not necessary.

Reply:
I think it would better reflect the hierarchy: ARM 15.1 Images is a derivative of the 15.1 Images, not of 15.0 Images. you're correct that because you're currently overwriting all implementations of 15.1 Images it doesn't make a lot of difference, but it will be more futureproof when 15.0 Image baseclass is going to be faded out.